techmatters/flex-plugins
# CHI-2319 cancel offline contacts

## Description

* Draft offline contacts get fully removed when cancelled
* In progress offline contacts get reloaded on browser session reloads

As an artefact of saving offline contacts under a single ID per user, 'draft offline contacts' may exist at unexpected times and cause confusion in search results, it might be best to just filter out all draft offline contacts from search results in HRM until the search experience is updated



### Checklist
- N/A Feature flags added
- N/A Strings are localized

### Related Issues
FixesCHI-2319

### Verification steps

* Cancel an offline contact, then start another - the data should all be reset to blanks / defaults
* Refresh the browser halfway through a contact, it should reload with the contact still in progress (minus and changes since you last changed tabs)
* Regression test offline contacts basic saving & searching
